examples/dependability/RBD/Holmakefile

```diff
@@ -0,0 +1,3 @@
+HOLDIR = ../../
+
+INCLUDES = $(HOLDIR)/miller/formalize
```

Comment:
This should surely be `$(HOLDIR)/examples/formalize`

Reply:
If it's still not clear to @ahmedwaqar, this means that the first line `HOLDIR = ../../` can be just deleted, because the variable `HOLDIR` will be setup by `Holmake` internally to the actual root directory of HOL4.

Reply:
Thanks. I'm aware of the ``HOLDIR`` environment variable and the HOL directory structure, but got clumsy to fix it before making a commit. I corrected the  path in ``RBD/Holmakefile`` to ``INCLUDES = $(HOLDIR)/examples/miller/formalize``.